cypress/e2e/Survey/index.spec.ts

```diff
@@ -0,0 +1,40 @@
+const surveyScreenTestIds = {
+  backButton: 'survey__back-button',
+  coverImage: 'survey__cover-image',
+  title: 'survey__title-text',
+  description: 'survey__description-text',
+  startSurveyButton: 'survey__start-survey-button',
+  loadingDialog: 'survey__loading-dialog',
+  toast: 'survey__toast',
```

Comment:
Do we need to assert those? if not, let's remove them all

Reply:
@nvminhtue removed 😄